activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ModelService.java

```diff
@@ -103,8 +111,28 @@ public ModelService(ModelRepository modelRepository,
                                          pageable);
     }
 
+    public Model createNewModelObject(String type,
+                                      String name) {
+      Model model = newModelInstance();
+      model.setType(type);
+      model.setName(name);
+      return model;
+    }
+
+    public Model newModelInstance() {
```

Comment:
can we improve the naming? does it create a model based on the instance type of model repository interface? I am not 100% sure tbh